dt/uni04delta-ipv6/kustomization.yaml

```diff
@@ -92,3 +92,188 @@ replacements:
           name: ironic
         fieldPaths:
           - spec.config
+
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.ironic.rpcTransport
+    targets:
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.ironic.template.rpcTransport
```

Comment:
I guess touching all these ironic values here is needed to avoid the OpenstackControlPlane update done to enable Ceph do not disable/change ironic config?

Reply:
Hmm. I don't remember changing anything with ironic. Must be some leftover after a merge conflict and I somehow missed it when going through the PR :thinking: . Thank you for revealing this, I'll delete this and retest with the downstream testproject.